magento-compatibility.js

```diff
@@ -4,6 +4,7 @@
 
 // PWA Studio version -> Magento version.
 module.exports = {
+    '8.0.0': '2.4.0 - 2.4.1',
```

Comment:
Should we also include `2.3.6`?

Reply:
so it should be `2.3.6 - 2.4.1`?

Reply:
I feel so but we need to get confirmation from either @jimbo or @dpatil-magento.